lib/OpenQA/Jobs/Constants.pm

```diff
@@ -156,3 +156,8 @@ sub meta_result {
     my ($result) = @_;
     return $META_RESULT_MAPPING{$result} // NONE;
 }
+
+sub is_ok_result {
+    my ($result) = @_;
+    grep { $result eq $_ } NOT_OK_RESULTS;
```

Comment:
```suggestion
    grep { $result eq $_ } OK_RESULTS;
```

And maybe use signatures right away (you don't have to convert the whole module).

Reply:
of course. But now – as expected – t/10-jobs.t fails